File: help/eng_gb/exercicios da ficha_11/Ficha11_009/Ficha11_009.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha11_009 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Height : " );
        int height = Integer.valueOf(keyb.nextLine());
        triangle(height ) ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void line( int n){
        for( int i = 0; i < n; i = i + 1 ) {
            System.out.print("*" );
        }
        System.out.print("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void triangle( int height){
        for( int i = 1; i < height + 1; i = i + 1 ) {
            line(i ) ;
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:23

File: help/eng_gb/exercicios da ficha_11/Ficha11_009/Ficha11_009.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Height : " ;
    $height = intval(isset($_POST["height"])? $_POST["height"] : isset($_GET["height"]) ? $_GET["height"] : 0);
    
    echo $height . "<br/>";// print input 
    triangle($height ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function line($n) {
    for( $i = 0; $i < $n; $i = $i + 1 ) {
        echo  "*" ;
    }
    echo  "\n" ;
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function triangle($height) {
    for( $i = 1; $i < $height + 1; $i = $i + 1 ) {
        line($i ) ;
    }
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 20:23

?>